src/main/java/customerproductorder/web/CustomerEditServlet.java

```diff
@@ -0,0 +1,53 @@
+package customerproductorder.web;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import customerproductorder.DataContainer;
+import customerproductorder.models.Customer;
+
+/**
+ * This class provide us using servlets.
+ */
+public class CustomerEditServlet extends HttpServlet {
+
+    /**
+     * This method is called when we send get to server.
+     */
+    private static final DataContainer container = DataContainer.instance;
+    //private int cardId;
+    private Customer editCustomer;
```

Comment:
You will have concurrent and multithreading issues here. Servlet is singleton in application. It means that you will use the same editCustomer for all requests and threads that serve request. So this field can be corrupted by multiple threads.